pages/popDetails/[id].js

```diff
@@ -0,0 +1,9 @@
+import PopDetails from "@/components/PopDetails";
+
+export default function PopDetailsPage() {
+  return (
+    <>
+      <PopDetails />
+    </>
```

Comment:
You don't need fragments here. You can drop them if you like.
```suggestion
    <PopDetails />
```